WALLET.md

```diff
@@ -119,6 +122,113 @@ If your machine does not recognize the `gem` command, you may need to [download
 
 And the app should be running in the simulator! If you run into any issues, see below for troubleshooting.
 
+#### Install on an M1
+
+Currently it is not possible to install the wallet natively. There are a few problems that need to be addressed before being able to run the repo on an M1:
+
+1. The M1 comes with a preinstalled version of Ruby that doesn't work with this repository.
+
+2. The build process that gets executed with `yarn dev:ios` is not able to finish with `nvm` installed.
+
+3. Running the necessary scripts with an M1-native node version will not work.
+
+4. It is currently not possible to run the repository in the integrated VSCode terminal.
+
+5. It is not possible to run the repository when the containing folder is located anywhere within `Documents` or `Desktop`.
+
+##### Solution
+
+1. Make sure all necessary software (VS Code, Terminal/iTerm2, XCode, Simulator) is running with Rosetta. You can verify this by typing `arch` in your terminal.
+
+```console
+$ arch
+-> i386
+```
+
+2. Reinstall `brew` with x86_64 architecture:
+
+```console
+arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+
+3. (optional) add your different brew installation paths as aliases to your `.zshrc`:
+
+```bash
+alias ibrew='arch -x86_64 /usr/local/bin/brew'
+alias mbrew='arch -arm64e /opt/homebrew/bin/brew'
+```
+
+4. Install [`rbenv`](https://github.com/rbenv/rbenv) on Intel architecture:
+
+```console
+ibrew install rbenv
+```
+4.1. Add rbenv initialization to your `.zshrc`:
+
+```bash
+eval "$(rbenv init - zsh)"
+```
```

Comment:
nit: `rbenv` detects the shell automatically when not provided:
```suggestion
4.1. Add rbenv initialization to your shell. For instance in `.zshrc` or `.bashrc`:

```bash
eval "$(rbenv init -)"
```
```